convert2rhel/utils.py

```diff
@@ -246,12 +246,21 @@ def ask_to_continue():
 
 
 def prompt_user(question, password=False):
+    """Prompt the user with a question and return his response to the caller.
+
+    :param question: The question to prompt to the user.
+    :param password: If the question is for a password, then the output is blanked out.
```

Comment:
@abadger did I make it right referencing the parameters this way?

Reply:
We don't use parameters too much in the code, but the style is correct! :+1: